travis.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/collective/buildout.plonetest/master/travis-5.x.cfg
```

Comment:
This was copied 1:1 from bobtemplates.plone which is considered best practice.
Breaking Plone 4 compatibility is no intention. This is WIP.